doc/rst/developer/bestPractices/Sanitizers.rst

```diff
@@ -15,28 +15,36 @@ require recompilation. Note that only GCC and Clang support sanitizers.
 How-to
 ------
 
-To use AddressSanitizer with Chapel (compiler and executables):
+To use AddressSanitizer with Chapel-generated executables only:
 
 .. code-block:: bash
 
      export CHPL_MEM=cstdlib
-     export CHPL_HOST_MEM=cstdlib
      export CHPL_TASKS=fifo
-     export CHPL_LLVM=none
-     export CHPL_SANITIZE=address
+     export CHPL_LLVM=none   # or set CHPL_TARGET_COMPILER=clang, gnu, etc.
```

Comment:
Recommend something like "# or set CHPL_TARGET_COMPILER to gnu or clang" (to prefer gnu since that's what we have nightly testing for and because at least historically only clang/gnu supported asan)